centml/values.yaml

```diff
@@ -1,20 +1,35 @@
 nodeSelector:
   region: LGA1
 
+affinity:
+  nodeAffinity:
+    requiredDuringSchedulingIgnoredDuringExecution:
+      nodeSelectorTerms:
+        - matchExpressions:
+            - key: topology.kubernetes.io/region
+              operator: In
+              values:
+                - "{{.Values.nodeSelector.region}}"
+            - key: gpu.nvidia.com/class
+              operator: In
+              values:
+                - A40
+                - A100_PCIE_40GB
+                - A100_PCIE_80GB
+                - Tesla_V100_NVLINK
```

Comment:
Does this chart have a vram requirement? I might want to make a couple adjustments to this list. We can def remove the v100 NVLINK since they are very low capacity.

Reply:
There is no VRAM requirement, I've removed V100